en/manage-ha-postgresql.md

```diff
@@ -0,0 +1,126 @@
+Title: PostgreSQL HA notes | MAAS
+
+
+# PostgreSQL HA: hot standby
+
+**Disclaimer: These are *notes* that the reader can peruse to get an idea of
+what may be involved at the command line level when setting up 'hot standby' HA
+mode in PostgreSQL. The below procedures are not regularly tested. Please use
+the
+[PostgreSQL documentation](https://www.postgresql.org/docs/9.5/static/high-availability.html)
+when setting up HA on PostgreSQL.**
+
+<!-- NOTES
```

Comment:
I don't really like storing this sort of info in the page, at least not while it remains unfiltered